views/recipe.handlebars

```diff
@@ -0,0 +1,18 @@
+<div class="recipe-form">
```

Comment:
This whole page does not exist in the most recent iteration so I would remove this from the PR.

Reply:
Got it.